src/Extensions/WorkflowApplicable.php

```diff
@@ -382,6 +403,12 @@ public function RecentWorkflowComment($limit = 10)
      */
     public function canPublish()
     {
+        // Ignore canPublish() if in the middle of having workflow added / removed
+        $key = $this->owner->baseClass() . '#' . $this->owner->ID;
+        if (isset(static::$itemsChangingWorkflow[$key])) {
+            return null;
```

Comment:
I feel like `false` would suit the method name better, assuming it should return a bool (unless false and null have different results for the user, like a validation error)

Reply:
They do, and this has been addressed in #345